src/main/java/com/example/pupbuddy/PupBuddyController.java

```diff
@@ -3,6 +3,12 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+
+/**
+ * Add Documentation here for code
+ * @author
+ */
```

Comment:
Empty JavaDoc isn't adding any value.  Better not to have it at all in that case, as it is just clutter.